experimental/LieAlgebras/test/root_systems-test.jl

```diff
@@ -0,0 +1,7 @@
+@testset "LieAlgebras.RootSystem" begin
+  R = RootSystem(:F, 4)
+  @test cartan_matrix(R) ==
+    matrix(QQ, 4, 4, [2, -1, 0, 0, -1, 2, -2, 0, 0, -1, 2, -1, 0, 0, -1, 2])
+  @test number_of_roots(R) == 48
+  @test number_of_roots("F4") == number_of_roots(R)
```

Comment:
```suggestion
  R = root_system(:F, 4)
  @test cartan_matrix(R) ==
    matrix(QQ, 4, 4, [2, -1, 0, 0, -1, 2, -2, 0, 0, -1, 2, -1, 0, 0, -1, 2])
  @test number_of_roots(R) == 48
  @test number_of_roots(:F, 4) == number_of_roots(R)
```
adapt to changed interfaces